src/index.js

```diff
@@ -1,35 +1,202 @@
+} else {
+  todoArray = JSON.parse(localStorage.getItem('todoList'));
+}
+
+const todoListRender = (desc, complete, indx) => {
+  const todoListItemContainer = document.createElement('div');
+  todoListItemContainer.classList.add('todo-element-padding');
+  todoListItemContainer.classList.add('todo-item-container');
+
+  const todoListCheck = document.createElement('input');
+  todoListCheck.setAttribute('type', 'checkbox');
+  todoListCheck.setAttribute('id', `todo-list-check-${indx}`);
+  todoListItemContainer.appendChild(todoListCheck);
+
+  const todoListElement = document.createElement('span');
+  todoListElement.classList.add('ellipsis');
+  todoListElement.innerText = desc;
+
+  todoListItemContainer.appendChild(todoListElement);
+
+  const threeDotIcon = document.createElement('span');
+  threeDotIcon.innerHTML = '&#8942;';
+  todoListItemContainer.appendChild(threeDotIcon);
+
+  document
+    .getElementById('todo-list-element')
+    .appendChild(todoListItemContainer);
+
+  document
+    .getElementById('todo-list-element')
+    .appendChild(document.createElement('hr'));
+
+  todoListCheck.addEventListener('click', () => {
+    if (todoListCheck.checked === true) {
+      todoListCheck.nextElementSibling.style.textDecoration = 'line-through';
+      todoListCheck.nextElementSibling.style.opacity = '0.5';
+
+      todoArray[indx].completed = true;
+    } else {
+      todoListCheck.nextElementSibling.style.textDecoration = 'none';
+      todoListCheck.nextElementSibling.style.opacity = '1';
+
+      todoArray[indx].completed = false;
+    }
+    localStorage.setItem('todoList', JSON.stringify(todoArray));
+  });
+};
+
+document.getElementById('todo-enter').addEventListener('click', () => {
+  const descriptionTodo = InputElementTodo.value;
+  InputElementTodo.value = '';
+  localStorage.setItem('input', InputElementTodo.value);
+  todoArray.push({
+    description: descriptionTodo,
     completed: false,
-    index: 1,
-  },
-];
+    index: todoArray.length,
+  });
+  localStorage.setItem('todoList', JSON.stringify(todoArray));
+  todoListRender(descriptionTodo, false, todoArray.length - 1);
+});
 
 todoArray.forEach((todo) => {
+  const todoListItemContainer = document.createElement('div');
+  todoListItemContainer.classList.add('todo-element-padding');
+  todoListItemContainer.classList.add('todo-item-container');
+
   const todoListCheck = document.createElement('input');
   todoListCheck.setAttribute('type', 'checkbox');
   todoListCheck.setAttribute('id', `todo-list-check-${todo.index}`);
-  todoListCheck.classList.add('todo-element-padding');
-  document.getElementById('todo-list-element').appendChild(todoListCheck);
+  todoListCheck.classList.add('test-check');
+  todoListItemContainer.appendChild(todoListCheck);
 
   const todoListElement = document.createElement('span');
-  todoListElement.classList.add('todo-element-padding');
+  todoListElement.classList.add('ellipsis');
   todoListElement.innerText = todo.description;
```

Comment:
- [ ] As required for the project, kindly ensure that you have created a **new JavaScript file** for the new functionality you added for this milestone. For example, you can create a file for `adding`, `removing`, and `editing` a task then import them in your `index.js` main file.